osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -206,10 +204,11 @@ protected override void CreateNestedHitObjects(CancellationToken cancellationTok
                         });
                         break;
 
-                    case SliderEventType.LegacyLastTick:
-                        // we need to use the LegacyLastTick here for compatibility reasons (difficulty).
-                        // it is *okay* to use this because the TailCircle is not used for any meaningful purpose in gameplay.
-                        // if this is to change, we should revisit this.
+                    case SliderEventType.LastTick:
+                        // Of note, we are directly mapping LastTick (instead of `SliderEventType.Tail`)  to SliderTailCircle.
+                        // It is required as difficulty calculation and gameplay relies on reading this value.
+                        // (although it is displayed in classic skins, which may be a concern).
```

Comment:
Not sure what this "displayed in classic skins" part means?

Reply:
Irrelevant as this comment is removed in the follow-up PR. But basically this mapped tick is visualised on classic skins: 
https://github.com/ppy/osu/blob/8e16b1d50784e62123d4ab308985b186e3c4ecb3/osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderTail.cs#L70